### PR TITLE
Fix tooltip positioning within elements with height or width of 0

### PR DESCRIPTION
**Pull request by batzlerg**
In a Backbone-style [TemplateView](https://www.npmjs.com/package/template-view), subviews can be rendered into an empty placeholder element (i.e. width=0, height=0) in the template. The logic in `getHalfDimensions` depends on the element's width and height dimensions, but because `0` (the value for both height and width in the placeholder element) is interpreted as falsy, `getHalfDimensions` returns `undefined` instead of the expected `0`.

**TL;DR:**

 `0 / 2 = 0` but if `element.width` or `element.height` is `0`, `getHalfDimensions` will return `undefined`

### CODE REVIEW

**Comment by gedas-monginas** [comment]
rushed a bit on merging, you still need to build JS file: `npm run build` its in readme.
and I guess this should be just a patch?

**Comment by batzlerg** [comment]
> rushed a bit on merging, you still need to build JS file: npm run build its in readme. and I guess this should be just a patch?

built and published dibs-react-tooltip@0.2.2
